lib/jbuild

```diff
@@ -0,0 +1,44 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags = function
+  | [] -> ""
+  | pkgs ->
+    let cmd = "ocamlfind ocamlc -verbose" ^ (
+        List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+      ) in
+    let ic = Unix.open_process_in
+        (cmd ^ " | grep -oEe '-ppx (\"([^\"\\]|\\.)+\"|\\w+)'")
+    in
+    let rec go ic acc =
+      try go ic (acc ^ " " ^ input_line ic) with End_of_file -> close_in ic; acc
+    in
+    go ic ""
+
+
+let coverage_rewriter =
+  let is_coverage = try Unix.getenv "BISECT_ENABLE" = "YES" with Not_found -> false in
```

Comment:
Is this step unnecessary with the latest bisect? https://github.com/aantron/bisect_ppx/blob/master/doc/advanced.md#with-jbuilder

Reply:
Or is it to only include `preprocess` as necessary, for extra safety?

Reply:
It is to avoid having to add `bisect_ppx` as a nonnecessary `opam` dependency